modules/test/playwright/tests/analytics-settings-web/utils/analyticsSettings.ts

```diff
@@ -100,26 +103,22 @@ export async function syncSite(page) {
 	});
 
 	await page
-		.getByTestId('Liferay DXP')
+		.getByTestId('Liferay')
 		.getByRole('button', {name: 'Assign'})
 		.click();
 
 	await page.getByRole('tab', {name: 'Sites'}).click();
 
-	await page
-		.getByTestId('1')
-		.getByTestId('Liferay DXP')
-		.getByLabel('')
-		.check();
+	await page.getByTestId('1').getByTestId('Liferay').getByLabel('').check();
```

Comment:
Hey @interaminense, same thing here. I don't think the default DXP site name will always be Liferay. I know that in Poshi on CI, it is not. This site corresponds to the name of the DXP instance which in the CI is created with the execution date being part of the instance name